### PR TITLE
Ts1 sub footer

### PR DESCRIPTION
**Pull request by Callum-Xavi**

### CODE REVIEW

**Review by KerryCX — APPROVED** [review]
Looks great Callum, I approve this footer :-)

File: my-app/src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    return (
+        <section>
+            <footer className="footer">
+                <div className="footer-content-centered">
+                    <h1 className="footer-title">Caaake, The Ultimate Online Movie Rental.</h1>
+                    <address className="footer-contact">
+                        <p>Email: <a href = "mailto: support@caaake.co.uk">support@caaake.co.uk</a></p>
+                        <a href="tel:+13115552368">(311) 555-2368</a>
+                        <p>100 Winchester Circle Los Gatos, CA 95032 United States</p>
```
Comment (asjoss) [inline my-app/src/components/Footer.jsx:13]: Love the fake address 😎

**Review by asjoss — APPROVED** [review]
Good stuff Callum; I approve!

**Comment by Callum-Xavi** [comment]
Thanks people, enjoy your weekend!

**Comment by Callum-Xavi** [comment]
Sorry, forgot to actually merge it before closing it! xD

**Pull request merged**